packages/react/src/components/FormLabel/FormLabel.js

```diff
@@ -24,9 +25,9 @@ const FormLabel = ({ className, children, id, ...other }) => {
   );
 
   return (
-    <label htmlFor={id} className={classNames} {...other}>
+    <Text as="label" htmlFor={id} className={classNames} {...other}>
```

Comment:
Same question here since it can be `PropTypes.node`

Reply:
Yep, will remove